app/src/main/java/org/jellyfin/mobile/player/ui/PlayerMenus.kt

```diff
@@ -225,10 +263,23 @@ class PlayerMenus(
         fragment.suppressControllerAutoHide(false)
     }
 
+    private fun formatBitrate(bitrate: Double): String {
+        val (value, unit) = when {
+            bitrate > BITRATE_MEGA_BIT -> bitrate / BITRATE_MEGA_BIT to " Mbps"
+            bitrate > BITRATE_KILO_BIT -> bitrate / BITRATE_KILO_BIT to " kbps"
+            else -> bitrate to " bps"
```

Comment:
Do we want to use (untranslateable-) strings for these?

Reply:
Don't think it's necessary.